content/blog/2017/06/2017-06-26-share-jenkins-world-keynote-stage.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Come Share the Jenkins World Keynote Stage with Me!"
+tags:
+- event
+- jenkinsworld
+author: Kohsuke Kawaguchi
```

Comment:
This isn't a valid author tag, the author entry should correspond to a file in `content/_data/authors`